### PR TITLE
use @defVariable for RoME Variable Types

### PR DESCRIPTION
**Pull request by Affie**
Just testing it for now. Needs DFG v0.10.2

### CODE REVIEW

**Comment by dehann** [comment]
towards #244

**Comment by dehann** [comment]
shall we target a minor release on RoME then, v0.10.0 is next.  This is really great!  Thanks for doing!

File: src/RoME.jl [hunk 1]
```diff
@@ -228,15 +228,17 @@ include("SpecialDefinitions.jl")
...
+# include("variables/DynPose2D.jl")
 
 # 3D
-include("variables/Point3D.jl")
-include("variables/Pose3D.jl")
+# include("variables/Point3D.jl")
+# include("variables/Pose3D.jl")
 
+#uses DFG v0.10.2 @defVariable for above
+include("variables/VariableTypes.jl")
```
Comment (Affie) [inline src/RoME.jl:241]: @dehann is it ok to have all variable definitions in one file like this?

**Comment by Affie** [comment]
Still to do is to update all the test that uses the old softtype `ut` fields.

File: src/RoME.jl (hunk 1 above)
Comment (dehann) [reply src/RoME.jl:241]: yeah, saw you did that -- think that will work well thanks otherwise they are all over the place -- good call!

**Comment by Affie** [comment]
Maybe convert on nstime from second, millisecond, float, etc.

**Comment by dehann** [comment]
this will cross paths with #352 sooner or later.

**Comment by GearsAD** [comment]
Awesome, thanks!

**Pull request merged**